### PR TITLE
Scroll util

### PR DESCRIPTION
**Pull request by flaurida**

### CODE REVIEW

File: tests/scroll.test.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    expect(await getScrollValue(page, elementHandle)).toEqual({
+      x: 0,
+      y: 2000,
+    });
+  });
+
+  it('does not throw error if can partially scroll element', async () => {
+    await page.goto(`${TEST_URL}large`);
+
+    await scroll(page, 'html', { x: 0, y: 2000, timeout: 1000 });
```
Comment (jperl) [inline tests/scroll.test.ts]: Can you add a test where you navigate while it is trying to scroll to make sure it does not throw an error. Like this test, but do not await the scroll promise, and do a goto.

Comment (flaurida) [reply tests/scroll.test.ts]: ok

Comment (jperl) [reply tests/scroll.test.ts]: Nevermind lets throw the error for now

Comment (jperl) [reply tests/scroll.test.ts]: Then I think the way to fix it is to add this first, and take the waitUntil options and pass them through (defaulting to load the same)

```
await page.waitForLoadState
```

**Pull request merged**